src/pages/Explorer/Category/Category.svelte

```diff
@@ -56,24 +111,68 @@
     padding: 0 var(--padding);
   }
 
+  .isMain .header {
+    border-radius: 8px;
+    border: 1px solid var(--porcelain);
+    margin-bottom: 24px;
+  }
+
   .icon {
     width: 32px;
     height: 32px;
     border-radius: 6px;
   }
 
-  .item,
-  .more {
+  .border .item,
+  .border .more {
     border-radius: 0;
     border-top: 1px solid var(--porcelain);
     --bg-hover: var(--athens);
   }
 
+  .center {
+    border: 1px solid var(--porcelain);
+    border-bottom: 0;
+  }
+
+  .more.last,
+  .isMain .item {
+    --bg-hover: var(--athens);
+  }
+
+  .isMain .item {
+    border-radius: 0;
+  }
+
   .item {
     padding: 12px var(--padding);
   }
 
   .more {
     padding: 13.5px 0;
   }
+
+  .postdate {
+    margin-bottom: 12px;
+    margin-left: 24px;
+  }
+
+  .isMain .first {
+    border: 1px solid var(--porcelain);
+    border-bottom: 0;
+    border-top-right-radius: 8px;
+    border-top-left-radius: 8px;
+  }
+
+  .isMain .last {
+    border: 1px solid var(--porcelain);
+    border-bottom-right-radius: 8px;
+    border-bottom-left-radius: 8px;
+    margin-bottom: 24px;
+  }
+
+  .isMain .more.last {
```

Comment:
For better composition `isMain` class should be passed via component's parent. This is relevant for the custom behaviour in a generic component